File: Tutorial/PyOpenCV/DLIP_Tutorial_OpenCV_Image.py
import cv2 as cv

# Read image
HGU_logo = 'C:\\Users\\skrua\\source\\repos\\DLIP\\Image\\HGU_logo.jpg'
src = cv.imread(HGU_logo)
src_gray = cv.imread(HGU_logo, cv.IMREAD_GRAYSCALE)

# Write image
HGU_logo_name = 'writeImage.jpg'
cv.imwrite(HGU_logo_name, src)

# Display image
cv.namedWindow('src', cv.WINDOW_AUTOSIZE)
cv.imshow('src', src)

cv.namedWindow('src_gray', cv.WINDOW_AUTOSIZE)
cv.imshow('src_gray', src_gray)

cv.waitKey(0)

File: Tutorial/DLIP_Tutorial_OpenCV_Image/DLIP_Tutorial_OpenCV_Image.cpp
#include <iostream>
#include <opencv2/opencv.hpp>

using namespace std;
using namespace cv;

int main()
{
	/*  read image  */
	String HGU_logo = "C:/Users/skrua/source/repos/DLIP/Image/HGU_logo.jpg";

	Mat src = imread(HGU_logo);

	Mat src_gray = imread(HGU_logo, 0);  // read in grayscale

	// Check for failure
	if (src.empty())
	{
		cout << "Could not open or find the imagesssss" << endl;
		cin.get(); //wait for any key press
		return -1;
	}


	/*  write image  */
	//String fileName = "writeImage.jpg";
	//imwrite(fileName, src);

	/*  display image  */
	namedWindow("src", WINDOW_AUTOSIZE);
	imshow("src", src);

	namedWindow("src_gray", WINDOW_AUTOSIZE);
	imshow("src_gray", src_gray);

	waitKey(0);
}